Content.Server/Botany/Systems/BotanySwabSystem.cs

```diff
@@ -19,7 +19,7 @@ public sealed class BotanySwabSystem : EntitySystem
     {
         [Dependency] private readonly DoAfterSystem _doAfterSystem = default!;
         [Dependency] private readonly PopupSystem _popupSystem = default!;
-        [Dependency] private readonly MutationSystem _mutationSystem = default!;
+        [Dependency] private readonly IEntityManager _entMan = default!;
```

Comment:
system dependencies are fine in other systems, just in the component was bad. this can be changed back, sorry if i was unclear

Reply:
Fixed, thanks.